integration/solana/simple.spec.ts

```diff
@@ -94,45 +96,60 @@ describe('Deploy solang contract and test', function () {
         expect(Number(res.result)).toBe(89);
         res = await contract.functions.op_u256(5, 123456789, 9, { simulate: true });
         expect(Number(res.result)).toBe(6662462759719942007440037531362779472290810125440036903063319585255179509);
-        res = await contract.functions.op_i256(6, 10000000000000, 8, { simulate: true });
+        res = await contract.functions.op_u256(6, 10000000000000, 8, { simulate: true });
         expect(Number(res.result)).toBe(2560000000000000);
-        res = await contract.functions.op_i256(7, 10000000000000, 8, { simulate: true });
+        res = await contract.functions.op_u256(7, 10000000000000, 8, { simulate: true });
```

Comment:
Why this change?

Reply:
You already test shift for signed integers [here](https://github.com/hyperledger/solang/blob/4ad568fff31e7dede49918d3eea01bc6d5224451/integration/solana/simple.spec.ts#L79-L83).

As all other tests in this set call the `u256` function, I thought you mistyped the function name. In addition, shift with unsigned integers will have no test without this change.